articles/ensemble-bias-var/index.md

```diff
@@ -0,0 +1,125 @@
+
+[Source](https://medium.com/ml-research-lab/ensemble-learning-relation-with-bias-and-variance-431cdc0a3fc9)
+
+### Bias Variance Trade-Off
+
+It is desirable to achieve low bias and variance to ensure accurate predictions. High bias and high variance hint at lower performance.
+
+![biasvariancetrade](/engineering-education/ensemble-bias-var/biasvariancetrade.png)
+
+*Bias-Variance Trade-off.*
+
+[Source](https://medium.com/ml-research-lab/ensemble-learning-relation-with-bias-and-variance-431cdc0a3fc9)
+
+Let’s interpret the image above.
+The center of the circles represents the actual value. The center of these circles depicts a model with perfect prediction ability. The crosses show the predicted value. The more we move from the center, the more the quality of prediction reduces.
+If predictions are concentrated in one area (which happens not to be the center), underfitting is present. This is because of high bias and low variance. The notable distance from the center of the circles is due to high bias. The crosses being so close to each other shows how variance.
+The scattering of predictions around the outer circles shows that overfitting is present. Low bias ensures the distance from the center of the circles is low. On the other hand, high variance is responsible for the crosses existing at a notable distance from each other.
+
+Increasing bias leads to a decrease in variance. Suppose we reduce bias, variance increases. An ideal model would have low variance and low bias. This is shown in the image below.
+
+![modelcomplexity](/engineering-education/ensemble-bias-var/modelcomplexity.png)
+
+*Model Complexity.*
+
+[Source](https://www.geeksforgeeks.org/ml-bias-variance-trade-off/?ref=rp)
+
+Ensemble learning is especially useful when trying to control the two errors.
+
+### Ensemble Methods, Bias and Variance
+
+We covered ensemble learning techniques like bagging, boosting, and stacking in a previous [article](/engineering-education/ensemble-learning/). As a result, we won’t reintroduce them here. We mentioned that bagging helps to reduce then variance while boosting reduces bias. In this section, we seek to understand how bagging and boosting impact variance and bias.
+
+#### Bagging and Variance
+
+Bagging is meant to reduce the variance without increasing the bias. This technique is especially effective where minute changes in a learner’s training set lead to huge changes in the predicted output. Bagging reduces the variance by aggregating individual models. These models have dissimilar statistical properties like means and standard deviations, among others.
+
+Let’s demonstrate how bagging reduces variance with a simple equation:
+
+• Assume we measure a random variable (x), with a normal distribution, which is denoted as N(µ,σ^2). µ is the mean of the distribution. It could also represent its median or mode. The parameter σ is the standard deviation.
```

Comment:
It would look neater if you used Latex for representing mathematical equations, either inline or block. Let me know if you need help with converting these equations into Latex.